pages/desktop/page_provider.py

```diff
@@ -18,21 +18,21 @@ def _set_window_size(self):
         if self.selenium.get_window_size()['width'] < 1920:
             self.selenium.set_window_size(1920, 1080)
 
-    def _go_to_page(self, page_object, do_login=False, user='default'):
+    def _go_to_page(self, page_object, username=None, password=None):
         self._set_window_size()
         self.selenium.get(self.base_url + page_object._page_url)
         page_object.is_the_current_page
-        if (do_login):
-            page_object.sign_in(user)
+        if all((username, password)):
+            page_object.sign_in(username, password)
         page_object.header.dismiss_staging_site_warning_if_present()
         return page_object
 
-    def _go_to_page_with_login_redirect(self, page_object, user='default'):
+    def _go_to_page_with_login_redirect(self, page_object, username, password):
```

Comment:
I must say I'm not a big fan of what's going on in this file, but I understand that this PR is about transitioning off `credentials` and not about refactoring the code.